arnab_bhadra/arnabbhadra_PostgresAssignment/src/Services/classMiddleware.ts

```diff
@@ -0,0 +1,29 @@
+import {page} from "../Utility/Page";
+import {validationPageAndPageSize} from "../Utility/Validation";
+import * as message from "../Utility/message";
+import * as koa from "koa";
+import * as uuid from 'uuid';
+import {ClassSchedule} from "../Entity/ClassSchedule";
+import * as classController from "../DBController/ClassController";
+
+export const insertClassScheduleInfo = async (ctx: koa.Context, next: koa.Next) => {
```

Comment:
this should be in controller, middleware usually contains some helper function that logs, authorises, error handlers etc